src/biz/bokhorst/xprivacy/XPrivacy.java

```diff
@@ -29,64 +28,107 @@
 	private static String mSecret = null;
 	private static List<String> mListHookError = new ArrayList<String>();
 	private static List<CRestriction> mListDisabled = new ArrayList<CRestriction>();
-
-	// http://developer.android.com/reference/android/Manifest.permission.html
-
-	static {
-		if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP && mListDisabled.size() == 0) {
```

Comment:
If you were wondering: I disabled this to prevent tripping SELinux (which could result in a bootloop) when reading the 'disabled' file on Lollipop (initZygote may not have access to /data/system on all devices).